packages/compass-components/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import ConfirmationModal from './components/confirmation-modal';
+import Icon from './components/icon';
 
-export { ConfirmationModal };
+export { ConfirmationModal, Icon };
```

Comment:
Just a personal preference, so feel free to ignore, but for components that we don't do anything additional with I think we can just re-export them without creating new files:

```suggestion
export { default as ConfirmationModal } from '@leafygreen-ui/confirmation-modal';
export { default as Icon } from '@leafygreen-ui/icon';
```